whats-new-in-tidb-4.0.md

```diff
@@ -69,22 +67,26 @@ TiUP is a new package manager tool introduced in v4.0 that is used to manage all
 
 + Transparent Data Encryption (TDE) is a new feature that provides protection for the entire database. This feature, when enabled, is transparent to applications that are connected to TiDB and does not require any change to the existing applications. Because this TDE feature operates at the file level, TiDB encrypts data before writing data to disk, and decrypts data before reading data from memory to ensure data security. Currently, the AES128-CTR, AES192-CTR, and AES256-CTR encryption algorithms are supported. You can manage keys via AWS KMS. See [Encryption at Rest](/encryption-at-rest.md) for details.
 
-### Backup and Restore
+## Backup and Restore
 
 Support the Backup & Restore (BR) feature to quickly back up and restore data of a single TiDB cluster to ensure data reliability and to meet enterprises’ needs of backup and restore or the graded protection of information security. Support quickly backing up and restoring all data or a certain range of sorted data. See [Backup & Restore](/br/backup-and-restore-tool.md) for details.
 
-### Service level features
+## Service level features
 
 + Support caching the execution plan of `Prepare` or `Execute` to improve the SQL execution efficiency.
 + Support self-adapting to the thread pool and streamlining the number of thread pools. Optimize the processing and scheduling of requests to improve product usability and performance.
 + The Follower Read feature refers to using any follower replica of a Region to serve a read request under the premise of strongly consistent reads. This feature improves the throughput of the TiDB cluster and reduces the load of the leader. It contains a series of load balancing mechanisms that offload TiKV read loads from the leader replica to the follower replica in a Region. TiKV's Follower Read implementation guarantees the consistency of data reading; combined with Snapshot Isolation in TiDB, this implementation provides users with strongly consistent reads. See [Follower Read](/follower-read.md) for details.
+
+## TiCDC
+
+TiCDC is a tool for replicating the incremental data of TiDB. This tool is implemented by pulling TiKV change logs, which ensures high reliability and availability of data. You can subscribe to the change information of data, and the system automatically sends data to the downstream. Currently, the downstream database must be MySQL compatible (such as MySQL and TiDB) or Kafka. You can also extend the supported downstream systems based on the [TiCDC Open Protocol](/ticdc/ticdc-open-protocol.md). See [TiCDC](/ticdc/ticdc-overview.md) for details.
```

Comment:
Oh, we support MySQL compatible database, Kafka and Pulsar now. We need to update the Chinese document too.

Reply:
Updated in https://github.com/pingcap/docs/pull/3754/commits/6107168f0205494b36b44279dd210852b5fdb726